packages/site/src/data/react/courses.ts

```diff
@@ -412,4 +412,16 @@ export const courses: Course<typeof courseTags[number]>[] = [
 		href: "https://themodern.dev/courses/build-a-fullstack-app-with-nextjs-supabase-and-prisma-322389284337222224",
 		tags: ["state management", "data fetching", "routing", "forms", "authentication", "hooks", "tailwind", "SSG"],
 	},
+	{
+		title: "Complete Intro to React, v7",
+		author: "Brian Holt",
+		image: "https://github.com/btholt.png",
+		description:
+			"Learn to build real-world applications using modern React! Much more than an intro, you’ll start from the ground up, getting to using the latest features in React, including hooks, effects, context, and portals. Throughout the course, you’ll piece together tools from the entire React ecosystem (like Parcel, ESLint, TailwindCSS, and React Router) to build a complete application to browse adoptable pets!",
+		paymentType: "paid",
+		level: "beginner",
+		format: "interactive",
```

Comment:
I think a better category would be video.

Reply:
Hello @jdwilkin4 I've submitted a new commit with the requested changes.